app/src/main/res/values-zh/strings.xml

```diff
@@ -393,22 +406,25 @@
     <string name="skip_on_error_description">当发生错误时跳过当前歌曲</string>
     <string name="skip_on_error_notification">由于错误，已跳过 %s</string>
     <string name="skip_on_error_notification_unknown_song">由于错误，已跳过当前歌曲</string>
-    <string name="reverb_small_room">Small room</string>
-    <string name="reverb_medium_room">Medium room</string>
-    <string name="reverb_large_room">Large room</string>
-    <string name="reverb_medium_hall">Medium hall</string>
-    <string name="reverb_large_hall">Large hall</string>
-    <string name="reverb_plate">Plate</string>
+    <string name="reverb_small_room">小房间</string>
+    <string name="reverb_medium_room">中等房间</string>
+    <string name="reverb_large_room">大房间</string>
+    <string name="reverb_medium_hall">中等大厅</string>
+    <string name="reverb_large_hall">大厅</string>
+    <string name="reverb_plate">板式混响</string>
 
     <plurals name="format_reset_blacklist_description">
+    	<item quantity="one">从黑名单中移除 %1$s 首歌曲</item>
         <item quantity="other">从黑名单中移除所有 %1$s 首歌曲</item>
     </plurals>
 
     <plurals name="format_reset_quick_picks_amount">
+    	<item quantity="one">删除 %1$s 次播放事件</item>
         <item quantity="other">删除 %1$s 次播放事件</item>
     </plurals>
 
     <plurals name="song_count_plural">
+    	<item quantity="one">%1$d 首歌曲</item>
```

Comment:
Why did you add these quantities? According to Lint these are redundant. Aren't they exactly the same?

Reply:
Ah I see - you're right, I mistakenly added those plural quantities when they weren't needed. Sorry for the confusion, I'll remove them in the next update.